app/models/authentication.rb

```diff
@@ -15,6 +15,13 @@ def self.new(*args, &block)
 
   belongs_to :resource, :polymorphic => true
 
+  has_many :authentication_configuration_script_bases,
+           :dependent  => :destroy,
+           :class_name => "AuthenticationConfigurationScriptBase"
```

Comment:
This shouldn't be needed anymore because it is fixed by https://github.com/ManageIQ/manageiq/pull/13632